scripts/deploy.py

```diff
@@ -160,9 +160,31 @@ def preprocess_release():
         new_assets_file.write(content)
 
 
+def check_dev_dependencies_are_up_to_date():
+    """Runs start.sh to ensure that dev dependencies are up-to-date."""
+
+    cmd = ['bash', 'scripts/start.sh']
+    if os.path.exists('start_output.txt'):
+        os.remove('start_output.txt')
+
+    with open('dev_output.txt', 'w') as out:
+        process = subprocess.Popen(cmd, stdout=out)
+
+        while True:
+            with open('dev_output.txt', 'r') as f:
+                lines = f.readlines()
+                for line in lines:
+                    if 'Setup complete!' in line:
```

Comment:
Do none of the dependencies use this string on completion of installation? (Maybe we can make it custom by saying 'Oppia setup complete!' or something like that.

Reply:
Done.